aws-stacks/ga4gh-testbed-dynamic-cft.yaml

```diff
@@ -0,0 +1,172 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Template for testbed backend instance and rds connection
+Parameters:
+  HostPort:
+    Type: String
+    Default: 4500
+  ParamContainerPort:
+    Type: String
+    Default: 4500
+  ApiImage:
+    Type: String
+    Default: ga4gh/ga4gh-testbed-api:test
+  VpcId:
+    Type: String
+    Default: vpc-0d772394df78b1311
+  SubnetA:
+    Type: String
+    Default: subnet-0be7a1fb3a96191b4
+  SubnetB:
+    Type: String
+    Default: subnet-0c465ec61b98bffac
+  SubnetC:
+    Type: String
+    Default: subnet-07b0e79f4d9bbf9d0
+  DBHost:
+    Type: String
+    Default: dev-ga4gh-testbed-database-database-bqdkifegcrpe.c0q4keadjybw.us-east-2.rds.amazonaws.com
+  DBName:
+    Type: String
+    Default: testbed_db
+  DBPort:
+    Type: String
+    Default: 5432
+  DBUser:
+    Type: String
+    Default: testbed_writer
+  DBPass:
+    Type: String
+  DBSchema:
+    Type: String
+    Default: public
+  TestbedLG:
+    Type: String
+    Default: testbed-backend-lg
+  Region:
+    Type: String
+    Default: us-east-2
+  Prefix:
+    Type: String
+    Default: ecs
+  ExecutionRole:
+    Type: String
+    Default: arn:aws:iam::282236276875:role/ECSFargateExecutionRole-TT
+  SecurityGroupId:
+    Type: String
+    Default: sg-036fa481aa113d84d
+
+Resources:
+
+  ALBBalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: TestbedAlbBalancer
+      Subnets:
+        - Ref: SubnetA
+        - Ref: SubnetB
+        - Ref: SubnetC
+      Type: application
+      SecurityGroups:
+      - !Ref SecurityGroupId
+
+  ALBTargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      Name: TestbedALBTargetGroup
+      VpcId:
+        Ref: VpcId
+      Protocol: HTTP
+      Port:
+        Ref: HostPort
+      HealthCheckEnabled: true
+      HealthCheckPath: /testbeds
+      HealthCheckIntervalSeconds: 120
+      HealthCheckTimeoutSeconds: 90
+      TargetType: ip
+
+  ALBListener:
+    Type: AWS::ElasticLoadBalancingV2::Listener
+    Properties:
+      DefaultActions:
+        - TargetGroupArn:
+            Ref: ALBTargetGroup
+          Type: forward
+      LoadBalancerArn:
+        Ref: ALBBalancer
+      Port:
+        Ref: HostPort
+      Protocol: HTTP
+
+  ECSCluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: dev-ga4gh-testbed-cluster
+
+  ECSService:
+    Type: AWS::ECS::Service
+    DependsOn: ALBListener
```

Comment:
Looks like it also depends on ECSCluster

Reply:
Added.